nailgun/entities.py

```diff
@@ -2774,6 +2778,17 @@ def __init__(self, server_config=None, **kwargs):
         self._meta = {'api_path': 'api/v2/ptables', 'server_modes': ('sat')}
         super(PartitionTable, self).__init__(server_config, **kwargs)
 
+    def update(self, fields=None):
+        """Fetch a complete set of attributes for this entity."""
```

Comment:
> API has confusing behavior sometimes - some entities require extra dict, some - don't, and some are working ok both with and without it.

Yeah. It's a real bugger.

Yes, please do remove the tests from `test_entities.py` as well.